jobs/cloud_controller_ng/spec

```diff
@@ -1282,6 +1285,8 @@ properties:
     description: "Use deprecated Thin webserver. Please note that when using Thin instead of Puma you miss out on the following benefits: Better resource utilization, well maintained and more performant. Thin will be removed in a future release. `cc.experimental.use_puma_webserver` takes precedence over this."
     default: false
 
+  cc.allow_user_creation_by_org_manager:
```

Comment:
No default value?

Reply:
Not needed, I would say. Benefit of this is that if it is not configured, it won't show up in the cloud_controller_ng.yml config. So we have a less bloated configuration.